Resources/Locale/en-US/objectives/round-end.ftl

```diff
@@ -11,7 +11,9 @@ objectives-player-named = [color=White]{$name}[/color]
 objectives-no-objectives = {$custody}{$title} was a {$agent}.
 objectives-with-objectives = {$custody}{$title} was a {$agent} who had the following objectives:
 
-objectives-objective-success = {$objective} | [color={$markupColor}]Success![/color]
+objectives-objective-success = {$objective} | [color={$markupColor}]Success![/color] ({$progress}%)
+objectives-objective-partial-success = {$objective} | [color={$markupColor}]Partial Success![/color] ({$progress}%)
+objectives-objective-partial-failure = {$objective} | [color={$markupColor}]Partial Failure![/color] ({$progress}%)
```

Comment:
why are you passing in the colors if they're just static strings in the code? just define them here.